webapps/src/main/webapp/javascript/automatic-translation/extensions.js

```diff
@@ -100,6 +100,17 @@
       },
     });
 
+    extensionRegistry.registerExtension('notes-automatic-translation', 'auto-translate', {
```

Comment:
you named the extension point "notes-automatic-translation". Does that mean we only add autotranslation extension into ? 
In fact, you named the extension point with a name which is related to the content of the extension, not of what is the extension point.

As the extension point is in notes, translation menu, I would name it "notes", with type 'translation-menu-extension'

Reply:
name of extension updated